echo-web/src/test/resources/echo-test.yml

```diff
@@ -7,6 +7,10 @@ front50:
 orca:
   baseUrl: 'http://localhost:8083'
 
+services:
```

Comment:
I'd rather use `@MockBean FiatService fiatService` in the failing tests since it means we're not making up a magic port number, and since it reduces the chance of network traffic, even to localhost.

Reply:
Update the code as suggested.